packages/@react-spectrum/list/src/styles.css

```diff
@@ -599,7 +658,54 @@
 
 .react-spectrum-ListView {
   &:focus-ring {
-    border-color: var(--spectrum-global-color-blue-500);
-    box-shadow: inset 0 0 0 1px var(--spectrum-table-cell-border-color-key-focus);
+    border-color: var(--spectrum-listview-border-color-key-focus);
+    box-shadow: inset 0 0 0 1px var(--spectrum-listview-border-color-key-focus);
   }
 }
+
+@media (forced-colors: active) {
+  .react-spectrum-ListView,
+  .react-spectrum-ListViewItem {
+    --spectrum-listview-background-color: Canvas;
+    --spectrum-listview-item-background-color: Canvas;
+    --spectrum-listview-item-background-color-selected: Canvas;
+    --spectrum-listview-border-color-key-focus: Highlight;
+    --spectrum-listview-item-border-color: CanvasText;
+    --spectrum-listview-item-border-color-selected: Highlight;
+    --spectrum-listview-item-border-color-key-focus: Highlight;
+    --spectrum-listview-row-sticky-focus-indicator-color: Highlight;
+    --spectrum-listview-droptarget-background-color: var(--spectrum-alias-global-color-transparent);
+    --spectrum-listview-dropindicator-border-color: Highlight;
+    --spectrum-listview-dropindicator-circle-border-color: Highlight;
+    --spectrum-listview-dropzone-border-color-selected-hover: Highlight;
+    --spectrum-listview-item-badge-text-color: HighlightText;
+    --spectrum-listview-item-badge-background-color: Highlight;
+    --spectrum-listview-item-title-text-color: CanvasText;
+    --spectrum-listview-item-description-text-color: CanvasText;
+  }
+
+  .react-spectrum-ListView-row {
+    &:focus-ring {
+      .react-spectrum-ListViewItem {
+        /* Adds a full outline style to the focused listview item to better distinguish between
+        the selected, not selected, selected + focused and selected + not focused
+        states without using a background color on the row in forced-colors: active mode. */
+        &:after {
+          outline: var(--spectrum-listview-row-sticky-focus-indicator-width) solid var(--spectrum-listview-row-sticky-focus-indicator-color);
+          outline-offset: calc(-1 * var(--spectrum-listview-row-sticky-focus-indicator-width));
+          top: calc(-1 * var(--spectrum-listview-border-width));
+          height: calc(100% + var(--spectrum-listview-border-width));
+        }
+      }
+    }
+  }
+
+  .react-spectrum-ListViewItem-draghandle-container {
+    .react-spectrum-ListViewItem-draghandle-button {
+      /* Use CanvasText, so that focus ring color on drag handle button matches
+      focus ring color for other interactive elements within the listview item. */
+      --spectrum-listview-item-border-color-key-focus: CanvasText;
```

Comment:
this should be a different variable that in non forced colors uses the same value as the item-border-color-key-focus, but with a different name would allow us to override it here in the top level
```
.react-spectrum-ListView {
  --spectrum-listview-item-draghandle-border-color-key-focus: var(--spectrum-listview-item-border-color-key-focus);
}

@media (forced-colors: active) {
  .react-spectrum-ListView {
    --spectrum-listview-item-draghandle-border-color-key-focus: CanvasText;
  }
}
```
then the comment won't be needed anymore